packages/client/hmi-client/src/components/drilldown/tera-drilldown-header.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+	<header>
+		<div class="title-row">
+			<div class="title-container">
+				<h4><slot /></h4>
+				<i v-if="props.tooltip" v-tooltip="tooltip" class="pi pi-info-circle" />
+			</div>
+			<Button
+				class="close-mask"
+				icon="pi pi-times"
+				text
+				rounded
+				aria-label="Close"
+				@click="emit('close')"
+			/>
+		</div>
+		<div class="tabs-row">
+			<div>
+				<TabView v-if="views.length > 1" :active-index="activeIndex" @tab-change="onTabChange">
+					<TabPanel v-for="(view, index) in views" :key="index" :header="view" />
+				</TabView>
+			</div>
+			<a target="_blank">Documentation</a>
```

Comment:
Add an optional `documentationURL` prop.

```suggestion
			<a :href="documentationURL" rel="noopener noreferrer">Documentation</a>
```